libs/cdk/src/lib/model.ts

```diff
@@ -85,6 +85,22 @@ export type RxStrategies<T extends string> = RxCustomStrategyCredentials<
   RxStrategyNames<T>
 >;
 
+export interface RxComponentInput<T, S> {
+  strategy?: RxStrategyNames<S> | Observable<RxStrategyNames<S>>;
+  renderCallback?: NextObserver<T>;
+  patchZone?: boolean;
```

Comment:
there is, but it doesn't serve the same purpose. It takes `patchZone: NgZone | false` instead of a boolean. Instead I have just turned the configuration Interface to a private type now. This way we don't have to specify any public naming for now